third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -207,6 +212,23 @@ The `google_sheets_options` block supports:
     that BigQuery will skip when reading the data. At least one of `range` or
     `skip_leading_rows` must be set.
 
+The `hive_partitioning_options` block supports:
+
+* `mode` (Optional) - When set, what mode of hive partitioning to use when
```

Comment:
These don't look great in the downstream markdown: https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-3335-old..auto-pr-3335?short_path=aa2f4fe#diff-aa2f4fe690d70a1d1d5a0f4bd663d428

Can we either add new lines before the `*` character, which should cause it to format as a list item, or quote `AUTO` and `STRINGS` in back ticks for formatting

Reply:
Additionally, it looks like `CUSTOM` is a valid mode here as well. It isn't explicitly called out in the docs which is strange, but I see references to it here: https://cloud.google.com/bigquery/docs/hive-partitioned-queries-gcs#setting_hivepartitioningoptions_using_the_rest_api